cmake/get_cpm.cmake

```diff
@@ -10,12 +10,23 @@ endif()
 
 # Expand relative path. This is important if the provided path contains a tilde (~)
 get_filename_component(CPM_DOWNLOAD_LOCATION ${CPM_DOWNLOAD_LOCATION} ABSOLUTE)
-if(NOT (EXISTS ${CPM_DOWNLOAD_LOCATION}))
+
+function(dl_cpm)
```

Comment:
Can we rename it to a more verbose function, e.g. `download_cpm`?

Reply:
yes sure (did that now)